ForumProject/subscriptions/views.py

```diff
@@ -1,3 +1,199 @@
-from django.shortcuts import render
+from rest_framework.response import Response
+from rest_framework import status, viewsets, permissions, pagination
+from rest_framework.decorators import action
+from .models import SubscribeInvestorStartup
+from .serializers import SubscribeInvestorStartupSerializer
+from users.models import UserInvestor
+from startups.models import Startup  
+from startups.serializers import StartupSerializer
+from django_filters.rest_framework import DjangoFilterBackend
+from startups.filters import StartupFilter
+from rest_framework import filters
+from rest_framework.permissions import IsAuthenticated
+from users.permissions import IsInvestorCompanySelected
+from .filters import MySubscriptionFilter
+from rest_framework.pagination import PageNumberPagination
 
 
+class CustomPagination(pagination.PageNumberPagination):
+    """
+    Custom pagination class for the list of startups.
+    """
+    page_size = 5
+    page_size_query_param = 'page_size'
+    max_page_size = 100
```

Comment:
Consider custom error handling for cases where a page number is invalid or exceeds bounds

Reply:
Done, thanks!